docs/tutorials/beginner_tutorial.rst

```diff
@@ -0,0 +1,210 @@
+=====================
+Start from the Basics
+=====================
+
+
+Installing orquestra-core
+=========================
+
+orquestra-core is a metapackage that installs other orquestra packages for you:
+
+.. code-block:: markdown
+
+    - orquestra-quantum
+    - orquestra-opt
+    - orquestra-vqa
+    - orquestra-qiskit
+    - orquestra-cirq
+    - orquestra-forest
+
+Installing orquestra-core can be done by running ``pip install orquestra-core``
+
+If instead you want to install from source, you can clone `this repo <https://github.com/zapatacomputing/orquestra-core>`_, navigate to it, and do
+``pip install .``
+
+
+Creating basic circuits
+============================
+
+.. _creating_circuits:
+
+Let's say you want to build a quantum circuit just to build a Bell State and measure it. As a quick reminder, a Bell State is a maximally entangled state with the most well-known form:
+
+.. math::
+    | \phi^+ \rangle = \frac{1}{\sqrt{2}}( |00\rangle + |11\rangle)
+
+
+This state can be created using a Hadamard gate and a CNOT gate:
+
+.. code-block:: text
+
+         ┌───┐     
+    q_0: ┤ H ├──■──
+         └───┘┌─┴─┐
+    q_1: ─────┤ X ├
+              └───┘
+
+Let's create a circuit that does this for us! Create a new file ``bell_state.py`` and follow along
+
+First, we need to import the necessary pieces from Orquestra Core (some of these won't be used here, but rather will be used in future parts of these tutorials):
```

Comment:
Ah you mean import only when we need the part rather than all-at-once?

Reply:
Yes – it makes it easier for the person using the docs to have interact and copy-paste.

Reply:
Okay that does make it slightly less nice on our end because instead of being able to pull whatever future imports there are straight out of that chunk in the source file, we have to specify for each little bit, but we can make it work :)